README.md

```diff
@@ -1,61 +1,63 @@
-# Postgres
+# Deis Postgres v2
 
-A PostgreSQL database for use in the [Deis](http://deis.io) open source PaaS.
+Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide a lightweight, easy and secure way to deploy your code to production.
 
-This Docker image is based on the official
-[postgres](https://registry.hub.docker.com/_/postgres/) image.
+For more information about the Deis workflow, please visit the main project page at https://github.com/deis/workflow.
 
-Master/slave replication and leader election code is largely inspired by [`compose/governor`](https://github.com/compose/governor).
+## Beta Status
 
-Please add any [issues](https://github.com/deis/postgres/issues) you find with this software.
+This Deis component is currently in beta status, and we welcome your input! If you have feedback, please submit an [issue][issues]. If you'd like to participate in development, please read the "Development" section below and submit a [pull request][prs].
 
+# About
 
-## Deploying
+This component is a PostgreSQL database for use in Kubernetes. It builds on the official [postgres](https://registry.hub.docker.com/_/postgres/) Docker image. While it's intended for use inside of the Deis open source [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service), it's flexible enough to be used as a standalone pod on any Kubernetes cluster or even as a standlone Docker container.
 
-To build a dev release of this image, you will also need your own registry, but DockerHub or
-[Quay](https://quay.io/) will do fine here. To build, run:
+# Development
 
-```bash
-$ export DEIS_REGISTRY=myregistry.com:5000
-$ make docker-build docker-push
-```
+The Deis project welcomes contributions from all developers. The high level process for development matches many other open source projects. See below for an outline.
 
-This will compile the Docker image and push it to your registry.
+- Fork this repository
+- Make your changes
+- Submit a [pull request][prs] (PR) to this repository with your changes, and unit tests whenever possible.
+- If your PR fixes any [issues][issues], make sure you write Fixes #1234 in your PR description (where #1234 is the number of the issue you're closing)
+- The Deis core contributors will review your code. After each of them sign off on your code, they'll label your PR with LGTM1 and LGTM2 (respectively). Once that happens, you may merge.
```

Comment:
non-maintainers cannot merge their own code. Can we re-word this to something along the lines of

> Once that happens, a contributor will merge it.